moveit_simple/include/moveit_simple/moveit_simple.h

```diff
@@ -352,4 +357,29 @@ class CartTrajectoryPoint : public TrajectoryPoint
   Eigen::Affine3d pose_;
 };
 
+class nullPointException: public std::runtime_error
```

Comment:
In my previous review, I identified 3 failures, which should have mapped to 3 exceptions.  There are 4.  Is there a reason you added one.  The `nullPointException` and the `noTrajNameException` are the same general error.  A `name` was passed to a method that isn't recognized.